v3/src/models/data/data-set-types.ts

```diff
@@ -42,6 +42,13 @@ export interface IAddCasesOptions {
   canonicalize?: boolean;
 }
 
+export interface IMoveItemsOptions {
+  // id of item before/after which to move items
+  // if not specified, items are moved to end
+  before?: string;
+  after?: string;
```

Comment:
It would have been fine to just support `before`, but nice to have `after` as well.